### ISSUE TITLE
Advice on Menu Direction

### ISSUE THREAD

**Issue by alnutile** [alnutile/cms#19]
@meetinghouse made some progress on the menu sorting interface
See image.
Is that along the lines of what you think we be a way users can sort menus?

![screenshot 2014-05-10 05 20 09](https://cloud.githubusercontent.com/assets/365385/2935697/79823fce-d824-11e3-8a68-cbdcda3bfa78.png)

**Comment by alnutile** [comment]
ps you can log in to the system and click Admin Menu to try sorting and clicking save.

**Comment by meetinghouse** [comment]
Al - I just looked at this with John. It looks okay. Is there another
option,  or are you just sort of checking in about it?

I was able to save a menu order change successfully, no errors.

Sincerely,

Katherine Johnson, Content Developer
The Custom Building & Restoration Trades
Directoryhttp://www.restorationtradesdirectory.com/
http://www.restorationtradesdirectory.com
(413) 773-8800

Published by:
Corbett Research Group https://mail.google.com/
"Building Websites that Work - - for the Building Artisans."
http://www.restorationtrades.com
